docs/resources.md

```diff
@@ -22,7 +22,7 @@ Note that the resource model currently prohibits over-committing resources; we w
 All resources have a _type_ that is identified by their _typename_ (a string, e.g., "memory").  Several resource types are predefined by Kubernetes (a full list is below), although only two will be supported at first: CPU and memory.  Users and system administrators can define their own resource types if they wish (e.g., Hadoop slots).
 
 A fully-qualified resource typename is constructed from a DNS-style _subdomain_, followed by a slash `/`, followed by a DNS Label.
-* The subdomain must conform to [RFC 1035 section 2.3.1 'subdomain' syntax](http://tools.ietf.org/html/rfc1035) / [RFC 1123](http://www.ietf.org/rfc/rfc1123.txt) (e.g., `kubernetes.io`, `myveryown.org`).
+* The subdomain must conform to [RFC 1035](http://tools.ietf.org/html/rfc1035) / [RFC 1123](http://www.ietf.org/rfc/rfc1123.txt) (e.g., `kubernetes.io`, `myveryown.org`).
```

Comment:
You are right, my memory was flawed.  1123 is defined in terms of 1035.
Sorry for the noise.

On Fri, Jan 23, 2015 at 2:59 PM, Vish Kannan <notifications@github.com>
wrote:

> In docs/resources.md
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/3774#discussion_r23486949>
> :
>
> > @@ -22,7 +22,7 @@ Note that the resource model currently prohibits over-committing resources; we w
> >  All resources have a _type_ that is identified by their _typename_ (a string, e.g., "memory").  Several resource types are predefined by Kubernetes (a full list is below), although only two will be supported at first: CPU and memory.  Users and system administrators can define their own resource types if they wish (e.g., Hadoop slots).
> >
> >  A fully-qualified resource typename is constructed from a DNS-style _subdomain_, followed by a slash `/`, followed by a DNS Label.
> > -* The subdomain must conform to [RFC 1035 section 2.3.1 'subdomain' syntax](http://tools.ietf.org/html/rfc1035) / [RFC 1123](http://www.ietf.org/rfc/rfc1123.txt) (e.g., `kubernetes.io`, `myveryown.org`).
> > +* The subdomain must conform to [RFC 1035](http://tools.ietf.org/html/rfc1035) / [RFC 1123](http://www.ietf.org/rfc/rfc1123.txt) (e.g., `kubernetes.io`, `myveryown.org`).
>
> The identifiers doc
> <https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/design/identifiers.md>
> also needs to be fixed in that case.
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/3774/files#r23486949>
> .
>